### PR TITLE
#19238 |  Meta descriptions for dotnet wpf

### PR DESCRIPTION
**Pull request by LauraKellerGitHub**
User Story 1657392
https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1657392

Adding meta descriptions to topics

### CODE REVIEW

File: docs/framework/wpf/data/how-to-implement-property-change-notification.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Implement Property Change Notification"
+description: Enable your binding target properties to automatically reflect the dynamic changes of the binding source in Windows Presentation Foundation (WPF).
```
Comment (adegeo) [inline docs/framework/wpf/data/how-to-implement-property-change-notification.md:3]: I think this is a bit clearer.
```suggestion
description: Enable your properties to automatically notify a binding source when the property value changes in Windows Presentation Foundation (WPF).
```

Is it possible for you to also edit the intro paragraph? It seems completely incomprehensible. It looks like there was some update at some point that merged two intros.

I think that intro should change to:

```
To support <xref:System.Windows.Data.BindingMode.OneWay> or <xref:System.Windows.Data.BindingMode.TwoWay> binding and enable your binding target properties to automatically reflect the changes of the binding source .......
```

File: docs/framework/wpf/data/how-to-specify-the-binding-source.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Specify the Binding Source"
+description: Find out about ways to specify the binding source through this example in the Windows Presentation Foundation (WPF).
```
Comment (adegeo) [inline docs/framework/wpf/data/how-to-specify-the-binding-source.md:3]: I think this is a bit clearer.
```suggestion
description: Learn how to specify the binding source through this example in the Windows Presentation Foundation (WPF).
```

**Review by adegeo — CHANGES_REQUESTED** [review]
Thank you for these changes! I left some small suggestions for you.

File: docs/framework/wpf/app-development/dialog-boxes-overview.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Dialog Boxes Overview"
+description: Learn about the varieties of dialog boxes in Windows Foundation Presentation (WPF). Find out about menu bars, status bars and tool bars.
```
[the remark this answers is not in the document]
Comment (LauraKellerGitHub) [reply docs/framework/wpf/app-development/dialog-boxes-overview.md]: There's a require for the character count to be between 100 and 160 characters. Will this 134-character description work?  Learn about the varieties of dialog boxes in Windows Foundation Presentation (WPF) that you can use to gather and display information.

**Comment by LauraKellerGitHub** [comment]
@adegeo 
Thanks for your review and feedback. I've changed four descriptions as shown.
The description for dialog-boxes-overview.md was too short according to our requirement of 100 to 160 characters. See what you think of the rewrite, which is 134 characters.
Thanks!

**Comment by adegeo** [comment]
Looks good, thanks!!!

**Review by adegeo — APPROVED** [review]

**Pull request merged**